js/src/index.js

```diff
@@ -777,11 +783,16 @@ class JupyterScatterView extends widgets.DOMWidgetView {
       return;
     }
 
-    if (!newSelection || !newSelection.length) {
-      this.scatterplot.deselect({ preventEvent: true });
-    } else {
-      this.scatterplot.select(newSelection, { preventEvent: true });
-    }
+    const selection = newSelection && newSelection.length > 0
```

Comment:
could use optional chaining here, e.g. `newSelection?.length > 0 ? newSelection : undefined`

Reply:
Thanks for pointing this out! I've updated several dev packages and enabled ES2020 so that ESLint can parse optional chaining